react/src/pages/EndpointListPage.tsx

```diff
@@ -205,10 +211,10 @@ const EndpointListPage: React.FC<PropsWithChildren> = ({ children }) => {
                   type: 'primary',
                 },
                 onOk: () => {
-                  setOptimisticDeletingId(row.endpoint_id);
+                  setOptimisticDeletingId(row?.endpoint_id);
```

Comment:
Same here, It can not be `undefined`

Reply:
https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2963/FR-17-feat-update-desired_session_count-field-name-to-replicas#comment-PRRC_kwDOCRTcws5yANWE